pages/about.html

```diff
@@ -5,8 +5,40 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles/styles.css" rel="stylesheet">
 </head>
 <body>
+  <div class="container-about">
+      <header class="cell cell-1">
+          <nav>
+              <ul>
+                <li><a href="file:///Users/eunice/Developer/projects/personal-portfolio-site/pages/portfolio.html"> my projects</a></li>
+                <li><a href= "file:///Users/eunice/Developer/projects/personal-portfolio-site/pages/index.html">my page</a></li>
+              </ul>
+            </nav>
+      </header>
+      <aside class="cell cell-2"><img src="file:///Users/eunice/Developer/projects/personal-portfolio-site/pages/images/about-me.jpeg" alt="about-personal-picture" /></aside>
```

Comment:
Similar to above use relative paths for images

```suggestion
      <aside class="cell cell-2"><img src="images/about-me.jpeg" alt="about-personal-picture" /></aside>
```